packages/core/src/clients/IVectorIndexClient.ts

```diff
@@ -1,21 +1,28 @@
 import {IVectorIndexControlClient} from '../internal/clients';
 import {
-  VectorAddItemBatch,
+  VectorUpsertItemBatch,
   VectorDeleteItemBatch,
   VectorSearch,
 } from '../messages/responses/vector';
 import {VectorIndexItem} from '../messages/vector-index';
 
+export const ALL = Symbol('ALL');
```

Comment:
```suggestion
export const ALL_METADATA = Symbol('ALL_METADATA');
```

Unless you have better ideas :-)